pkg/reconciler/apiserversource/resources/receive_adapter.go

```diff
@@ -151,5 +153,9 @@ func makeEnv(args *ReceiveAdapterArgs) ([]corev1.EnvVar, error) {
 	}, {
 		Name:  "K_LOGGING_CONFIG",
 		Value: args.LoggingConfig,
-	}}, nil
+	},
+		{
+			Name:  "K_TRACING_CONFIG",
+			Value: args.TracingConfig,
+		}}, nil
```

Comment:
```go
	},
}, nil
```

Reply:
Tried to change as you say, but go fmt is wacky :smile: look at it now